compiler/rustc_mir_build/src/build/expr/as_place.rs

```diff
@@ -199,7 +205,7 @@ fn to_upvars_resolved_place_builder<'a, 'tcx>(
     from_builder: PlaceBuilder<'tcx>,
     tcx: TyCtxt<'tcx>,
     typeck_results: &'a ty::TypeckResults<'tcx>,
-) -> Result<PlaceBuilder<'tcx>, HirId> {
```

Comment:
Can't comment on the correct line, but the function docs need update around what err contains